bsip-00XX Dynamic market fees.md

```diff
@@ -0,0 +1,183 @@
+    BSIP: TBD
+    Title: Dynamic Market Fees
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-12-17
+    Updated: 2018-12-17
+    Discussion: 
+    Worker: TBD
+    
+# Abstract
+UIA owners do not have many marketing tools to increase trade volume of their assets.
+
+One of the ways to increase the trade volume can be Dynamic Market Fees that will provide additional discounts for active traders.
+For example, to set lower fees for trading larger volumes.
+
+So, instead of market fee, an asset owner can optionally set up dynamic_market_fee_percent property that may look like a following example table.
```

Comment:
Please elaborate. Since the given table is only meant as an example, how is that property to be set up and evaluated, and which degrees of freedom does this allow?

Reply:
That still doesnt define what the degrees of freedom are